pylint/extensions/_check_docs_utils.py

```diff
@@ -58,10 +58,19 @@ def possible_exc_types(node):
     """
     excs = []
     if isinstance(node.exc, astroid.Name):
-        excs = [node.exc.name]
+        inferred = safe_infer(node.exc)
+        if inferred:
```

Comment:
Sure thing. So if it's uninferable or None, we should just not report a warning at all rather than using the old behavior of reporting a warning for node.exc.name?